apps/roam/package.json

```diff
@@ -69,7 +69,7 @@
     "react-draggable": "4.4.5",
     "react-in-viewport": "1.0.0-alpha.20",
     "react-vertical-timeline-component": "3.5.2",
-    "roamjs-components": "0.85.4",
+    "roamjs-components": "0.85.6",
```

Comment:
i left these changes here after trying out the `autoSelectFirstOption` props because I think we should upgrade roamjs-compoenents anyway